community/community_extensions.json

```diff
@@ -268,5 +268,17 @@
         "description": "An extension of UpSet.js to create interactive set visualizations for more than three sets.",
         "source_code": "",
         "hostedTrexFileName": "UpSetjs.trex"
-    }
+    },
+    {
+        "name": "MultiSelect Parameter",
+        "website": "https://biplovkarna017.github.io/tableau-multiselect-parameter/",
+        "author": "Biplov Karna",
+        "github_username": "biplovkarna017",
+        "tags": [
+            "v_1.0"
+        ],
+        "description": "Concatenates selected values from a filter and assigns the result, delimited by '|', to a parameter.",
+        "source_code": "https://github.com/biplovkarna017/tableau-multiselect-parameter",
+        "hostedTrexFileName": "MultiSelectParameter.trex"
+    },
```

Comment:
Not sure if it would cause problems, but JSON generally won't have a trailing comma

Reply:
Thanks, Andy. I'll remove it. Looks like the old file didn't have that trailing comma.